packages/app/components/TokenIcon.tsx

```diff
@@ -1,35 +1,44 @@
 import { isAddress } from "viem";
 import { cva } from "class-variance-authority";
 import Image from "next/image";
-import { useNetwork } from "wagmi";
 import { twMerge } from "tailwind-merge";
 import { useTokenListContext } from "@/context/TokenListContext";
 import { Token } from "@/models/token/types";
+import { useState } from "react";
 
 interface TokenIconProps {
-  token: Token;
   className?: string;
   size?: "lg" | "md" | "sm" | "xs";
+  token: Token;
 }
 
-export const TokenIcon = ({ token, className, size }: TokenIconProps) => {
+export const TokenIcon = ({ className, size, token }: TokenIconProps) => {
```

Comment:
curious, do you have an extension that orders props? Might be interesting for me as well.

Reply:
I don't, I just order them manually. But I guess we could explore some linter to handle that on save D: